apps/src/lib/ui/PopUpMenu.jsx

```diff
@@ -46,6 +47,37 @@ class PopUpMenu extends Component {
     }).isRequired,
     children: PropTypes.any,
     className: PropTypes.string,
+    isOpen: PropTypes.bool,
+    beforeClose: PropTypes.func,
+  };
+
+  render() {
+    return (
+      <Portal
+        closeOnEsc
+        closeOnOutsideClick
+        isOpened={this.props.isOpen}
+        beforeClose={this.props.beforeClose}
+      >
+        <RadiumMenuBubble
+          targetPoint={this.props.targetPoint}
+          className={this.props.className}
+          children={this.props.children}
+        />
+      </Portal>
+    );
+  }
+}
+export default PopUpMenu;
+
+class MenuBubble extends Component {
```

Comment:
It gets even better. Assuming you don't have eslint warnings for react components without a display name, you can just use an anonymous class:

```jsx
export const MenuBubble = Radium(class extends Component {
  ...
});
```